classes/helper.php

```diff
@@ -41,6 +41,11 @@ class helper {
      */
     private static $bootstrapped = false;
 
+    /**
+     * @var boolean Have we sent the feature policy headers already?
+     */
+    private static $featurebootstrapped = false;
```

Comment:
do you need this? headers are queued and by default they replace previously set headers until they get flushed

```
header ( string $header [, bool $replace = TRUE
```

Reply:
Yeah its not exactly necessary. Its slightly more efficient and matches the CSP header behaviour of only being sent once no matter what, but I can remove it. These headers should remain the same in pretty much every case.